lib/Controller/RoomController.php

```diff
@@ -119,7 +119,8 @@ public function update(
 		bool $listenOnly,
 		bool $mediaCheck,
 		bool $cleanLayout,
-		bool $joinMuted
+		bool $joinMuted,
+		string $logoutURL
```

Comment:
We need some type of validation for the `logoutURL`. At least it has to start with `http://` or `https://`.

Reply:
This validation should also be done on the server side, otherwise an attacker could bypass the client logic.